persistence-jdbc/scaladsl/src/multi-jvm/resources/reference.conf

```diff
@@ -17,4 +18,18 @@ lagom.persistence.read-side {
     # additional random delay is based on this factor
     random-factor = 0.2
   }
+=======
+
+
+# Aggressively restart and ensure the cluster-distributed, readside actors so tests don't take that long.
+lagom.persistence {
+  read-side.failure-exponential-backoff {
+    min = 1 s
+    max = 2 s
+    random-factor = 0.2
+  }
+  cluster.distribution {
+    ensure-active-interval = 4s
+  }
+>>>>>>> master
```

Comment:
did we committed a file with conflicts?

Reply:
yes, fixed in d1c071d0e33001bd1b3ebba32ef1cde4fcf1084a